projects/storefrontlib/src/shared/services/trigger/config/trigger-config.ts

```diff
@@ -0,0 +1,22 @@
+import { OutletPosition } from '../../../../cms-structure/index';
+
+export abstract class TriggerConfig {
+  trigger?: {
+    asm?: TriggerOutletMapping | TriggerInlineMapping | TriggerUrlMapping;
+    skipLinks?: TriggerOutletMapping | TriggerInlineMapping | TriggerUrlMapping;
+  };
+}
+
+export interface TriggerOutletMapping {
```

Comment:
other ideas:
- render a component in the global message box (I know... ;) )
- render a component (message) in the service worker notification center

Reply:
but we can add that later